assignments/resume/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Ahmed Okasha</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab|Titillium+Web|Ubuntu" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link href="main.css" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+        <header class="row">
+            <section id="title" class="col-md-5 col-sm-12">
+                <h1 id="name">Ahmed Okasha</h1>
+                <b>Software Developer</b>
+                <p id="intro">A determined and dependable individual with a strong desire to learn and work hard, Dedicated to continuously
+                    developing, implementing, and adopting new technologies to maximize development efficiency I am looking
+                    for new challenges to gain further experience in programming and contribute to a company as a respectable
+                    team player.
+                </p>
+            </section>
+            <div id="pic-container" class="col-md-3 col-sm-6">
+                <img id="profile-pic" src="pic.jpg">
+            </div>
+
+            <ul id="contacts" align="right" class="col-md-4 col-sm-6">
+                <li>a.okasha.azzam@gmail.com <img src="message.png"></li>
+                <li>+2-0100-466-4076 <img src="phone.png"></li>
+                <li><a href="www.github.com/okashoon">www.github.com/okashoon</a> <img src="git.png"></li>
+                <li><a href="www.linkedin.com/in/okasha ">www.linkedin.com/in/okasha</a> <img src="linkedin.png"></li>
+            </ul>
+
+
+        </header>
+
+        <main class="row">
+            <div id="left" class="col-md-6 col-sm-6">
+                <h2>Work Experience</h2>
+                <section class="work">
+                    <time>01/2016-Present</time><span>Cairo,Egypt</span>
```

Comment:
when time is not machine readable you need to add datetime attribute in the machine readable format.